bitbots_dynamic_kick/src/kick_ik.cpp

```diff
@@ -21,7 +21,7 @@ void KickIK::reset() {
    * since we are using a gradient based method. Otherwise, the
    * first step will be not correct */
   std::vector<std::string> names_vec = {"LHipPitch", "LKnee", "LAnklePitch", "RHipPitch", "RKnee", "RAnklePitch"};
-  std::vector<double> pos_vec = {0.7, -1.0, -0.4, -0.7, 1.0, 0.4};
+  std::vector<double> pos_vec = {0.7, 1.0, -0.4, -0.7, -1.0, 0.4};
```

Comment:
I this part of this pull request?

Reply:
this is a fix to adapt kick to new robot model (knee joints changed and therefore the sign changed). was not supposed to be in this PR but needs to be on the master never the less. I would just keep it here, to spare additional work for another PR.